src/App.scss

```diff
@@ -0,0 +1,111 @@
+.Landing {
+  display: flex;
+  flex: 1;
+  flex-wrap: wrap;
+  height: 100vh;
+}
+
+.Login {
```

Comment:
It would be wise so start splitting your scss code into multiple files and import them all there after. This file will get crowded fast.